pkg/files/api.go

```diff
@@ -0,0 +1,89 @@
+package files
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/appgate/sdp-api-client-go/api/v17/openapi"
+	"github.com/appgate/sdpctl/pkg/configuration"
+)
+
+type FilesAPI struct {
+	Config     *configuration.Config
+	HTTPClient *http.Client
+}
+
+func (f *FilesAPI) List(ctx context.Context) ([]openapi.File, error) {
```

Comment:
Duplicate function of https://github.com/appgate/sdpctl/blob/b16e0cfd4f9c03290f891eacaae83decfb7734e1/pkg/appliance/api.go#L190-L196